types/mui-datatables/index.d.ts

```diff
@@ -193,8 +190,8 @@ export interface MUIDataTableOptions {
     rowHover?: boolean;
     rowsPerPage?: number;
     rowsPerPageOptions?: number[];
-    rowsSelected?: any[];
     rowsExpanded?: any[];
+    rowsSelected?: any[];
```

Comment:
Why is this re-ordered?

Reply:
Alphabetized properties inside interface